Assets/Scenes/SampleScene.unity

```diff
@@ -123,6 +123,63 @@ NavMeshSettings:
     debug:
```

Comment:
Need to change SampleScene name

Reply:
Will resolve in another push. Need to see if this breaks anything. It Shouldnt though.